generators/client/templates/react/src/main/webapp/robots.txt

```diff
@@ -4,9 +4,7 @@ User-agent: *
 Disallow: /api/account
 Disallow: /api/account/change-password
 Disallow: /api/account/sessions
-<%_ if (databaseType !== 'no' && databaseType !== 'cassandra') ( _%>
```

Comment:
The error was `(` instead of `{`
I didn't see it in my previous comment because I was on my phone